app/src/main/java/ciserver/ContinuousIntegrationServer.java

```diff
@@ -18,6 +19,7 @@
 public class ContinuousIntegrationServer extends AbstractHandler {
 
     Gson gson = new Gson();
```

Comment:
Can this also be final? Considering it should never change. Not terribly important